src/PrestaShopBundle/Form/Admin/Sell/Product/SEO/RedirectOptionType.php

```diff
@@ -198,8 +206,12 @@ public function configureOptions(OptionsResolver $resolver)
     private function getRedirectionAlertMessages(): array
     {
         return [
-            $this->trans('No redirection (404) = Do not redirect anywhere and display a 404 "Not Found" page.', 'Admin.Catalog.Help'),
-            $this->trans('No redirection (410) = Do not redirect anywhere and display a 410 "Gone" page.', 'Admin.Catalog.Help'),
+            $this->trans('Default = As specified in the Prestashop settings.', 'Admin.Catalog.Help'),
```

Comment:
I think this one needs wording modified?
Get rid of "=" sign. And the "Prestashop settings" sounds too abstract, should mention the actual related settings page like "ShopParameters > Traffic & SEO" (or wherever the default is set) or so... ping @Julievrz

Reply:
Well, the = sign was in the original help texts, I followed the example - `Permanent redirection (301) = Permanently display another product or category instead.`

What about `Default = As specified in Shop Parameters > Product settings.`

Reply:
sounds good to me :man_shrugging: Lets wait for wording experts on this one :v:

Reply:
@l-delin @Julievrz We need you! 😄

Reply:
Hello, we reviewed the wording 🔧👏 @Hlavtox